pkg/apis/servicecatalog/types.go

```diff
@@ -266,10 +266,16 @@ type InstanceSpec struct {
 	// provisioned from.
 	PlanName string
 
-	// Parameters is a YAML representation of the properties to be
+	// Parameters is set of properties to be
 	// passed to the underlying broker.
 	Parameters *runtime.RawExtension
 
+	// Optional: AlphaSecretParameters is reference to the secret object containing
+	// sensitive information to pass to the underlying broker. This may be
+	// empty if no secret object is specified.
+	// +optional
+	AlphaSecretParameters *v1.LocalObjectReference
```

Comment:
I'm surprised to see this suggestion after all the discussions we had in https://github.com/kubernetes-incubator/service-catalog/issues/621 :)

Also, it's different from what you have already implemented in Binding: Binding has a secret name, and all the credentials are stored to it as key-blob values. That's exactly what I do there too, but in the reverse direction.
See https://github.com/kubernetes-incubator/service-catalog/blob/master/pkg/controller/controller_binding.go#L358